addons/addon-base-raas/packages/base-raas-services/lib/study/__tests__/study-service.test.js

```diff
@@ -1294,4 +1294,103 @@ describe('studyService', () => {
       expect(retVal).toEqual(false);
     });
   });
+
+  describe('updatePermissions', () => {
```

Comment:
Could you add another use-case for using the `*` outside of the MyStudy context

Reply:
There should not be a use-case for using `*` outside of the MyStudy context.

Reply:
Removed the part that relies on `*` as migration indicator and disallowed `*` to be part of the update request so there is no need to test anymore